pkg/crosscluster/logical/logical_replication_job_test.go

```diff
@@ -180,6 +180,58 @@ func TestLogicalStreamIngestionJobNameResolution(t *testing.T) {
 	}
 }
 
+func TestOptimsitcInsertCorruption(t *testing.T) {
```

Comment:
nit: i'd add a comment that this a regression test.

Reply:
Done